recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
I really cannot understand this $PREFIX story...

How should I edit my `build.sh` and `test.sh` files? Thanks